frontend/src/shared/form/autocomplete-one-input.vue

```diff
@@ -183,9 +227,23 @@ export default {
       }
     },
 
+    async onScroll() {
+      // If limit is higher or equal to total number of members, do not fetch for more options
+      if (this.limit >= this.total) {
+        return
+      }
+
+      this.limit =
+        this.limit + AUTOCOMPLETE_SERVER_FETCH_SIZE
```

Comment:
One question, it looks to me like we are increasing the limit instead of keeping it constant, but using pagination. Is that correct? 

If so, why? 😀

Reply:
Yes that's what I'm doing. Unfortunately, from my understanding,  the `member/autocomplete` endpoint that we are using for this component only handles the `limit` parameter, not the `offset`. It also only retrieves the results and not the pagination. So I had to handle it this way
Does it not work like this?

Reply:
So right now we have lazy loading in the dropdown. We start with a limit of 20, and as the user scrolls down we just increase the limit